apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/resources/TraceResourceClient.java

```diff
@@ -140,4 +146,17 @@ public void updateTrace(UUID id, TraceUpdate traceUpdate, String apiKey, String
             assertThat(actualResponse.hasEntity()).isFalse();
         }
     }
+
+    public Map<String, Long> aggregateSpansUsage(List<Span> spans) {
```

Comment:
I'm not sure it belongs in the test client. Why not in a test utils class?

Reply:
Done in https://github.com/comet-ml/opik/pull/894/commits/6430973c43e0de665b136aefa3c1e0e87b9d5173